google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java

```diff
@@ -60,22 +60,32 @@ public void mouseClicked(MouseEvent event) {
   }
 
   void initTableModel() {
-    List<Blob> blobs = getBlobsAtLevel("");
+    List<Blob> blobs = getBlobsStartingWith("");
     if (!blobs.isEmpty()) {
       tableModel = new GcsBlobTableModel();
       tableModel.setDataVector(blobs, "");
       bucketContentTable.setModel(tableModel);
     }
   }
 
-  private void updateTableModel(String prefix) {
+  void updateTableModel(String prefix) {
     tableModel.setRowCount(0);
-    tableModel.setDataVector(getBlobsAtLevel(prefix), prefix);
+    tableModel.setDataVector(getBlobsStartingWith(prefix), prefix);
     tableModel.fireTableDataChanged();
   }
 
+  /**
+   * In the GCS backend there are no directories, just blobs with names. Implicit in the name,
```

Comment:
nit: This seems like a supporting paragraph that should exist under a summary fragment for this method. Consider adding a summary fragment like:

```java
/**
 * Returns the list of {@link Blob Blobs} that start with the given prefix.
 *
 * <p>In the GCS backend...
 */
```

Reply:
done